tests/functional/api.php

```diff
@@ -11,7 +11,7 @@
 
 // Launch PHP dev server in the background
 chdir(INSTALL_ROOT);
-exec('php -S localhost:8082 -t web/ > /dev/null 2>&1 & echo $!', $output);
+exec('./start.sh -tests-server > /dev/null 2>&1 & echo $!', $output);
```

Comment:
This is not going to work, when you kill the process at the end of the script, you will kill the shell process  now, not the PHP server opened in the background (you can check that you get an error message about a process that can't be killed when you do start.sh -tests and you can check in your browser that localhost:8083 does work.

At the end of the script you kill the process like that:
```bash
+exec('pkill -P ' . $pid);
```